.gitignore

```diff
@@ -10,3 +10,5 @@ pom.xml.asc
 *DS_Store
 push
 /.clj-kondo/.cache
+/.clj-kondo/manifold/manifold
```

Comment:
You were knee-deep in kondo recently. Are these not for sharing with other developers on Manifold?

Reply:
Nope, the thing is that when you run a full clj-kondo lint, it copies the config.edn files of the dependencies to .clj-kondo. The problem is that .clj-kondo copies the dependencies of the project itself under `manifold/manifold`, which brings a duplicate of the existing config.edn file.
![image](https://user-images.githubusercontent.com/55855728/224557191-ecfc9bf7-709b-4a6c-88b7-f670fa080f48.png)